src/main/java/org/folio/service/caches/MappingParametersCache.java

```diff
@@ -145,6 +206,115 @@ private List<Future<OrganizationCollection>> getOrganizationCollectionFutures(Ma
     return organizationCollectionFutures;
   }
 
+  private Future<List<Mtype>> getMaterialTypes(OkapiConnectionParams params) {
+    String materialTypesUrl = "/material-types?limit=" + settingsLimit;
```

Comment:
using StringBuilder is more effecient for add operations

Reply:
I think using StringBuilder will not make code more effective, but will also make it harder to read

Reply:
since using small concatenate, make sense to keep as it was